packages/jupyter-ai-magics/jupyter_ai_magics/magics.py

```diff
@@ -5,12 +5,14 @@
 import warnings
 from typing import Optional
 
+import click
```

Comment:
Is there a convention for sorting imports? I've been in the habit of grouping imports by their level, so `import foo` and `import bar` are grouped, as are `import baz.quux` and `import baz.quuux`. I also like to sort imports alphabetically within a level.

Reply:
Probably, but any lint changes are best addressed in a separate PR due to the merge conflicts they create.

Reply:
I believe the `isort` step in the `precommit` configuration other Jupyter projects use takes care of sorting imports. However, I don't like the default of using alphabetical sort. I prefer having two groups separated by newlines: one for standard library imports (e.g. `typing`, `os`, etc.) and another for local/third-party imports (e.g. `langchain`, `jupyter_ai_magics.providers`, etc.). I find that this is much more helpful for "indexing" all of the imports in a module as a developer. Let me know your thoughts on this.